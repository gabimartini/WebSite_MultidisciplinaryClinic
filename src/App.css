.App {
  text-align: center;
}

.App-logo {
  height: 40vmin;
  pointer-events: none;
}

@media (prefers-reduced-motion: no-preference) {
  .App-logo {
    animation: App-logo-spin infinite 20s linear;
  }
}

.App-header {
  background-color: #282c34;
  min-height: 100vh;
  display: flex;
  flex-direction: column;
  align-items: center;
  justify-content: center;
  font-size: calc(10px + 2vmin);
  color: white;
}

.App-link {
  color: #61dafb;
}

@keyframes App-logo-spin {
  from {
    transform: rotate(0deg);
  }
  to {
    transform: rotate(360deg);
  }
}

/* ---- */

#preloader {
  position:fixed;
  top:0;
  left:0;
  right:0;
  bottom:0;
  background-color: #eeecec80; 
  z-index:999; 
}
#preloader .inner {
  position: absolute;
  top: 50%;
  left: 50%;
  transform: translate(-50%, -50%);  
}
.bolas > div {
display: inline-block;
background-color: #E49468;
width: 25px;
height: 25px;
border-radius: 100%;
margin: 3px;
-webkit-animation-fill-mode: both;
animation-fill-mode: both;
animation-name: animarBola;
animation-timing-function: linear;
animation-iteration-count: infinite;
 
}
.bolas > div:nth-child(1) {
  animation-duration:0.75s ;
  animation-delay: 0;
}
.bolas > div:nth-child(2) {
  animation-duration: 0.75s ;
  animation-delay: 0.12s;
}
.bolas > div:nth-child(3) {
  animation-duration: 0.75s  ;
  animation-delay: 0.24s;
}

@keyframes animarBola {
0% {
  -webkit-transform: scale(1);
  transform: scale(1);
  opacity: 1;
}
16% {
  -webkit-transform: scale(0.1);
  transform: scale(0.1);
  opacity: 0.7;
}
33% {
  -webkit-transform: scale(1);
  transform: scale(1);
  opacity: 1; 
} 
}
