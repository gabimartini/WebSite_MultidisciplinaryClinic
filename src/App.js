import React from 'react';
import './App.css'
import { Route, Switch } from 'react-router-dom';
import Home from './Component/Home/Home'
import About from './Component/About/About'
import Contact from './Component/Contact/Contact'
import Layout from './Content/Layout'


function App() {
  return (
    <div>
<Layout>
   <Switch>
      <Route path="/" exact component={Home} />
      <Route path="/Sobre" component={About} />
      <Route path="/Contato" component={Contact} />
    </Switch>
  </Layout>
    </div>
    
  );
}

export default App;
