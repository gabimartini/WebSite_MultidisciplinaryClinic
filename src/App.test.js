import React from 'react';
import {BrowserRouter} from 'react-router-dom'
import * as renderer from 'react-test-renderer';
import App from './App';


test('App snapshot test', () => {
  const component = renderer.create(<BrowserRouter><App /></BrowserRouter>);
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
