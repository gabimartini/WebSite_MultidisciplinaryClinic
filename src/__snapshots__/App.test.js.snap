// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App snapshot test 1`] = `
<div>
  <div
    className="Main-Content"
  >
    <div
      className="Main_Menu"
    >
      <div
        className="DrawerToggle"
        onClick={[Function]}
      >
        <div />
        <div />
        <div />
      </div>
      <div
        className="Desktop"
      >
        <div
          className="Main_Items"
        >
          <li
            className="Dropbtn"
          />
          <div
            className="Desktop_Menu"
          >
            <li
              className="Menu_Item"
            >
              <a
                aria-current="page"
                className="active"
                href="/"
                onClick={[Function]}
                style={Object {}}
              >
                Home
              </a>
            </li>
            <li
              className="Menu_Item"
            >
              <a
                aria-current={null}
                href="/Sobre"
                onClick={[Function]}
              >
                Sobre Nós
              </a>
            </li>
            <li
              className="Menu_Item"
            >
              <a
                aria-current={null}
                href="/Contato"
                onClick={[Function]}
              >
                Contato
              </a>
            </li>
          </div>
        </div>
      </div>
    </div>
    <div>
      <div
        className={
          Array [
            "SideDrawer Close",
          ]
        }
      >
        <nav
          onClick={[Function]}
        >
          <div
            className="Mobile"
          >
            <li
              className="Menu_Item"
            >
              <a
                aria-current="page"
                className="active"
                href="/"
                onClick={[Function]}
                style={Object {}}
              >
                Home
              </a>
            </li>
            <li
              className="Menu_Item"
            >
              <a
                aria-current={null}
                href="/Sobre"
                onClick={[Function]}
              >
                Sobre Nós
              </a>
            </li>
            <li
              className="Menu_Item"
            >
              <a
                aria-current={null}
                href="/Contato"
                onClick={[Function]}
              >
                Contato
              </a>
            </li>
          </div>
        </nav>
      </div>
    </div>
    <main
      className="Content"
    >
      <div
        className="Main-Content"
      >
        <div
          className="main-head"
        >
          <div
            className="background-head"
          >
             
          </div>
          <div
            className="title-border"
          >
             
          </div>
          <p
            className="title"
          >
            Equipe Multidisciplinar
          </p>
          <div
            className="background-img"
          />
          <img
            alt=""
            className="img"
            src="equipe-cinza.png"
          />
        </div>
        <div
          className="Services-Main"
        >
          <div
            className="Title-Main"
          >
            <p
              className="Services-Title"
            >
              Nossos Serviços
            </p>
            <p
              className="Services-Sub-Title"
            >
              Nossa Equipe trabalha de forma humanizada
            </p>
          </div>
          <div
            className="Service-Box"
          >
            <div
              className="Service Mobile-One"
            >
              <span>
                <div
                  className="Title-Area"
                >
                  Nutrição
                </div>
              </span>
              <p
                className="Info-Area"
              >
                ⏵Funcional 
                <br />
                ⏵Alimentação 
                <br />
                Comportamental 
                <br />
                ⏵Nutrição Infantil
              </p>
            </div>
            <div
              className="Service Second-Color Mobile-Two"
            >
              <span>
                <div
                  className="Title-Area"
                >
                  Psicologia
                </div>
              </span>
              <p
                className="Info-Area"
              >
                ⏵Funcional 
                <br />
                ⏵Alimentação 
                <br />
                 Comportamental 
                <br />
                ⏵Nutrição Infantil
              </p>
            </div>
            <div
              className="Service Mobile-Three"
            >
              <span>
                <div
                  className="Title-Area"
                >
                  Medicina
                </div>
              </span>
              <p
                className="Info-Area"
              >
                ⏵Funcional 
                <br />
                ⏵Alimentação 
                <br />
                Comportamental 
                <br />
                ⏵Nutrição Infantil
              </p>
            </div>
            <div
              className="Service Second-Color Mobile-Four"
            >
              <span>
                <div
                  className="Title-Area"
                >
                  Fisioterapia
                </div>
              </span>
              <p
                className="Info-Area"
              >
                ⏵Funcional 
                <br />
                ⏵Alimentação 
                <br />
                Comportamental 
                <br />
                ⏵Nutrição Infantil
              </p>
            </div>
          </div>
        </div>
      </div>
    </main>
    <footer
      className="footer"
    >
      <p
        className="Adress"
      >
        <span>
          Sabe onde nos encontrar?
        </span>
         
        <br />
        R. Alm. Carlos da Silveira Carneiro, 182 - 
        <br />
        Agronômica, Florianópolis - SC, 88025-350, Brasil
      </p>
      <p
        className="Apointment"
      >
        <span>
          Nosso horário de atendimento:
        </span>
         
        <br />
        Segunda á sexta: 
        <br />
         das 8:00 às 18:00
      </p>
      <p
        className="Phone"
      >
        <span>
          Quer marcar um horário por telefone?
        </span>
         
        <br />
        048 12345678
      </p>
      <div
        className="social"
      >
        <a
          className="fa fa-linkedin lkdin"
          href="https://www.linkedin.com/"
          rel="noopener noreferrer"
          target="_blank"
        >
           
        </a>
        <a
          className="fa fa-facebook fb"
          href="https://www.linkedin.com/"
          rel="noopener noreferrer"
          target="_blank"
        >
           
        </a>
        <a
          className="fa fa-instagram ig"
          href="https://www.linkedin.com/"
          rel="noopener noreferrer"
          target="_blank"
        >
           
        </a>
      </div>
      <div
        className="copy"
      >
        <p>
          Copyright ©2020 All rights reserved by Gabriela Martini
        </p>
      </div>
    </footer>
  </div>
</div>
`;
